esmvaltool/recipes/hydrology/recipe_hydro_forcing.yml

```diff
@@ -16,6 +16,9 @@ documentation:
     - smeets_stef
     - aerts_jerom
 
+  maintainer:
+    - unmaintained
```

Comment:
@SarahAlidoost @Peter9192 Would one of you be interested in being the maintainer for the recipes that were developed to support the [eWaterCycle](https://ewatercycle.readthedocs.io/en/latest/) platform?